### PR TITLE
Contributing.md and contributing guide

### PR DESCRIPTION
**Pull request by bvhme**
A template for a basic Amsterdam contributing guide.

Also for the 'guides' section of the site.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### 5. Celebrate
+
+Your ideas, documentation and code have become an integral part of this project. You are the Open Source hero we need.
+
+We typically respond to new pull requests within a few business days.
+
+---
+
+For more information oh how to use and contribute to this project, please read the [README](README.md).
```
Comment (RRMoelker) [inline CONTRIBUTING.md]: typo

**Comment by RRMoelker** [comment]
Het is denk ik prettig als je een stukje markdown geeft die in de Readme.md van elk project staat die verwijst naar dit document.

Wat meer structuur in de Readme.md files zou zowiezo helpen met het sneller op gang komen in een project.

**Comment by MauriceHendriks** [comment]
Als jullie toch amsterdam.github.io aan het 'kapen' zijn, dan bemoei ik me er maar gelijk even mee :wink: Waarom niet gewoon alles in restructuredText? Github kan daar ook prima mee overweg.

**Comment by bvhme** [comment]
@kerkuil GitHub pages' native templating engine only eats MarkDown, which means that when a `.md` in the master gets updated the site automatically gets built and updated by GitHub. 

I hope that by keeping the tools necessary to contribute as limited as possible I can get as many people as possible from inside Gemeente Amsterdam to contribute, even if they are on the policy or management side. Now it is just a question of editing a file in GitHub's built-in file editor.

**Comment by bvhme** [comment]
@RRMoelker There are two ways we can go here, namely:
A: Add a link to every readme
B: Add a `CONTRIBUTING.md` to every repo

The advantage of B is that it will show up as a message in GitHub's interface when making an issue/PR and that it is copied with forking. The advantage of A is that is easier to edit the code.

Food for thought.

(Also I've fixed the typo!)

**Comment by MauriceHendriks** [comment]
> @kerkuil GitHub pages' native templating engine only eats MarkDown, which means that when a .md in the master gets updated the site automatically gets built and updated by GitHub.

Fair enough :+1:

**Comment by bvhme** [comment]
Suggestions from @Lytrix 

- [ ] Half in size, too many suggestions and things mentioned twice.
- [ ] Link to existing best practices and guides
- [ ] Inspiration: http://developer.tidepool.org/

**Comment by bvhme** [comment]
I've cut some fat and added a link to the GDS guidelines on committing.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+If you want to add to the documentation or code of one of our projects you should make a Pull Request. If you never used GitHub, get up to speed with [Understanding the GitHub Flow](https://guides.github.com/introduction/flow/).
+
+### 1. Make your changes
+
+Add your changes in commits [with a message that explains them](https://github.com/alphagov/styleguides/blob/master/git.md#commit-messages). Document choices or decisions you make in the commit message, this will enable everyone to be informed of your choices in the future.
+
+If you are adding code, make sure it adheres to the project's code and documentation style guide before you submit your pull request. You can read more about how to document and style your code in the project's [`README`](README.md).
+
+### 2. Pull Request
```
Comment (Vegter) [inline CONTRIBUTING.md]: I propose to add a sentence about tests. Write tests that show the behaviour of your newly added or changed code

**Review by Vegter — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (bvhme) [reply CONTRIBUTING.md]: Fixed!

Comment (bvhme) [reply CONTRIBUTING.md]: Does this solve the problem @Vegter

Comment (Vegter) [reply CONTRIBUTING.md]: I think so. Although I still feel uncertain about having projects without tests. As you mentioned, this project should be an example of a project without any automated tests. But should it not include a spell check for instance? What is a project without inherent automated quality assurance?

**Review by Vegter — APPROVED** [review]

**Pull request merged**